src/App.js

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
[OPTIONAL] Rather than disable the Linters, strive to fix them as this makes you more accustomed to the style guide of the tech.